### PR TITLE
Increased spacing between Contributions' subtext and the 3d personal activity image

### PR DESCRIPTION
**Pull request by somekindofwallflower**
Small fix, increase the space between text and image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 # Contributions
 (in the last 365 days, languages pie based on number of commits)
+
```
Comment (formidablae) [inline README.md:16]: Nice touch

**Review by formidablae — APPROVED** [review]

**Comment by formidablae** [comment]
Thank you for your proposed change.

**Comment by formidablae** [comment]
Closes #10

**Comment by somekindofwallflower** [comment]
> Thank you @somekindofwallflower for the proposed change.

You're welcome :)

**Pull request merged**